### PR TITLE
Revamp HTTP BBEs

### PR DESCRIPTION
**Pull request by TharmiganK**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/http-service-cache-response/http_service_cache_response.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# HTTP service - Sending cache response
+# REST service - Sending cache response
```
Comment (shafreenAnfar) [inline examples/http-service-cache-response/http_service_cache_response.md:1]: Send cache response

Comment (TharmiganK) [inline examples/http-service-cache-response/http_service_cache_response.md:1]:
```suggestion
# REST service - Send cache response
```

File: examples/index.json [hunk 2]
```diff
@@ -1449,6 +1449,22 @@
...
+            {
+                "name": "Send different status codes with payload",
+                "url": "http-send-different-status-codes-with-payload",
+                "verifyBuild": true,
+                "verifyOutput": false,
+                "disablePlayground": true,
+                "isLearnByExample": false
+            },
+            {
+                "name": "Sending cache response",
```
Comment (TharmiganK) [inline examples/index.json:1462]:
```suggestion
                "name": "Send cache response",
```

File: examples/http-service-cache-response/http_service_cache_response.md (hunk 1 above)
Comment (TharmiganK) [reply examples/http-service-cache-response/http_service_cache_response.md]: Fixed via [4ac3adc](https://github.com/ballerina-platform/ballerina-distribution/pull/3705/commits/4ac3adc401cf65be70dc60250d746e3f4c0d445f)

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**